src/experiments.js

```diff
@@ -73,7 +73,9 @@ export function isDevChannelVersionDoNotUse_(win) {
  * @return {boolean}
  */
 export function isExperimentOn(win, experimentId) {
-  return getExperimentIds(win).indexOf(experimentId) != -1;
+  let cookieFlag = getExperimentIds(win).indexOf(experimentId) != -1;
```

Comment:
No, not quite. We need a slightly different process:

1. If experiment has been read before - return its value.
2. If experiment define by the cookie, save and return its value of `true` (it can't be `false` in this case).
3. If `AMP_CONFIG[experimentId]` exists, evaluate, save and return its value as following:
 - `AMP_CONFIG[experimentId] <= 0` -> `experiment = false`
 - `AMP_CONFIG[experimentId] >= 1` -> `experiment = true`
 - `experiment = Math.random() < AMP_CONFIG[experimentId]`
4. Save and return `false`.